### PR TITLE
fix: enhance feed layout responsiveness

### PR DESCRIPTION
**Pull request by RitaDee**
## Description

https://github.com/open-sauced/insights/pull/1490#pullrequestreview-1565363428

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/insights/assets/106407013/fe12355d-6a4d-4bd0-b334-895d69e10740

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
@RitaDee `npm run format` will fix lint errors.

**Comment by RitaDee** [comment]
@brandonroberts, I have resolved the conflict.

Thank you, @bdougie for the lint error fix.

**Comment by bdougie** [comment]
bug: Long names in top contributors mess up spacing on small screens.

I noticed a weird side effect. I did see this get reported elsewhere. OK to have it addressed on another PR

My suggestion is to add truncation when the width is adjusted.


Screen Shot 2023-08-07 at 4 46 56 PM

**Comment by babblebey** [comment]
> bug: Long names in top contributors mess up spacing on small screens.
> 

Ha, this had been noted already and it was the next issue @RitaDee planned on raising, thank you for pointing that out again.

**Review by babblebey — COMMENTED** [review]
LG 🍕

File: pages/feed/index.tsx [hunk 1]
```diff
@@ -281,7 +284,8 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
             <FollowingHighlightWrapper selectedFilter={selectedRepo} emojis={emojis} />
           </TabsContent>
         </Tabs>
-        <div className="md:hidden gap-6 mt-10 md:flex-1 lg:flex md:flex-col">
+        
+        <div className="hidden lg:w-[24%] w-[30%] gap-6 mt-10 flex-none lg:flex flex-col">
```
Comment (NsdHSO) [inline pages/feed/index.tsx:288]: @RitaDee  it is not a good practice to use `%` we can't use `w-1/3` -> width: `33.33...%`  `w-4` -> `25%`????

Comment (RitaDee) [reply pages/feed/index.tsx:288]: @NsdHSO noted with thanks. I will fix that.

**Comment by bdougie** [comment]
Thanks for jumping on this @RitaDee

**Review by brandonroberts — APPROVED** [review]
Changes visually look good 👍  . Needs one more round of merge conflicts resolved

**Comment by RitaDee** [comment]
@brandonroberts, I will fix it.

**Comment by RitaDee** [comment]
@brandonroberts, I have fixed the conflict.

**Comment by bdougie** [comment]
I am seeing nothing on mobile currently image

**Comment by brandonroberts** [comment]
@RitaDee that last merge broke the UI according to the PR preview. Will you take a look?

**Comment by RitaDee** [comment]
@brandonroberts I will take a look at it.

**Comment by RitaDee** [comment]
@brandonroberts, I have this fix on a new branch. 
Will it be okay if I close this PR?

**Comment by RitaDee** [comment]
@brandonroberts, I have this fix on a new branch. 

Here is the link: 
https://github.com/open-sauced/insights/pull/1538

I will close this PR.

**Pull request closed without merging**